changelog/7760.breaking.rst

```diff
@@ -0,0 +1,2 @@
+`sunpy.map.GenericMap.contour` now uses ContourPy instead of scikit-image for faster performance and consistency with `sunpy.map.GenericMap.draw_contour`, reducing reliance on scikit-image.
```

Comment:
<unknown>:7: WARNING: py:obj reference target not found: sunpy.map.GenericMap.draw_contour [ref.obj]

Reply:
Its fixed now